### ISSUE TITLE
Nit: Add PR url to LAND_STACK_TIP 

### ISSUE THREAD

**Issue by drisspg** [modularml/stack-pr#50]
# Summary
Curious if you would be open to it. I often create a PR or a stack of PRs and have a hard finding the stack of URls. Would you be open to adding this or making it an option/ configurable?

**Comment by ZolotukhinM** [comment]
Yeah, that would be a nice change! It does print it in the logs (at least if `--verbose` is set), but having it printed in a summary at the end would be helpful. Please feel free to send a PR if you're up to it :)

**Comment by AmaranthineCodices** [comment]
As a workaround, if your terminal supports the [terminal hyperlinks spec](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) (a lot of them do), running `stack-pr view` after you create the stack will give you clickable links.
